addons/website/views/assets.xml

```diff
@@ -89,6 +77,16 @@
     </xpath>
 </template>
 
+<template id="assets_frontend_user_custom" inherit_id="portal.assets_frontend" name="Website Assets user custom" priority="55">
+    <xpath expr="//link[last()]" position="after">
+        <!-- Custom empty file for user custom rules -->
+        <link rel="stylesheet" type="text/scss" href="/website/static/src/scss/user_custom_rules.scss"/>
+        <!-- Custom empty file for user custom bootstrap override -->
+        <link rel="stylesheet" type="text/scss" href="/website/static/src/scss/user_custom_bootstrap_overridden.scss"/>
+        <link rel="stylesheet" type="text/scss" href="/website/static/src/scss/bootstrap_overridden.scss"/>
```

Comment:
Review the explanation I gave you before, those should stay in _assets_frontend_helpers otherwise they do not work. Test your branch, it cannot work if you put those in assets_frontend: edit the file with the example given inside and it should still work.